test/schema/asset_uploads/create_asset_request_mutation.js

```diff
@@ -43,6 +43,7 @@ describe("addAssetToConsignmentSubmission", () => {
         }),
     }
 
+    expect.assertions(1)
```

Comment:
This ensures that at least 1 `expect` is called in the test - /cc @alloy who was looking for something like this back when we moved this project to Jest